### DISCUSSION TITLE
ambiente virtuale door e nuove librerie

### DISCUSSION THREAD

**Discussion by fabiodelogu** [c-hydro/door#11] [General]
@c-hydro/team-door bisognerebbe creare un ambiente virtuale specifico per door, facendo un censimento
delle librerie utilizzate fino ad ora.
in aggiunta per i prodotti h141 si sono rese necessarie le librerie eccodes e l'applicazione di cdo che in qualche
modo dovremmo aggiungere nella configurazione iniziale dell'ambiente virtuale

**Comment by AndreaLibe** [comment]
Ciao Fa, sia cdo che eccodes credo che fossero già usate da gsmap che dovrebbe essere pure lui in grib, non ci sono nella vecchia hyde?

**Comment by AndreaLibe** [comment]
Oltre alle "tradizionali" (xarray, numpy, pandas) quelle un po' più unusual che ho censito sono:
cdsapi
tempfile
urllib
cdo
siphon
netrc
gdal
ftplib
ftputil
bs4
requests
rioxarray

**Reply by FrancescoAvanzi** [reply]
@AndreaLibe, tu avresti annotato per caso come installare cdo e l'ordine delle librerie di sistema annesse? Così magari editiamo il ReadMe e rimane traccia

**Comment by AndreaLibe** [comment]
Si ho usato questo installer che ti allego.
Installa tutto a parte le HDF5 e le ZLIB di cui chiede il path.

[install_CDO.sh_.txt](https://github.com/c-hydro/door/files/11158915/install_CDO.sh_.txt)

**Reply by FrancescoAvanzi** [reply]
@AndreaLibe possiamo metterlo su GitHub anche lui? Almeno ce lo abbiamo da usare

**Reply by AndreaLibe** [reply]
Yes non era rigorosisismo nella formattazione, per quello non avevo mai osato :)

**Reply by FrancescoAvanzi** [reply]
si lo stiamo studiando

**Reply by FrancescoAvanzi** [reply]
ci lavora @fabiodelogu
